Dockerfile

```diff
@@ -1,32 +1,41 @@
-FROM ocaml/opam:alpine-3.20-ocaml-5.2 as build
+FROM ocaml/opam:alpine-3.20-ocaml-5.2 AS build
 
 # Install system dependencies
-RUN sudo apk update && sudo apk add --update libev-dev openssl-dev gmp-dev oniguruma-dev inotify-tools curl-dev autoconf
+RUN sudo apk -U upgrade --no-cache && sudo apk add --no-cache \
+    autoconf \
+    curl-dev \
+    gmp-dev \
+    inotify-tools \
+    libev-dev \
+    oniguruma-dev \
+    openssl-dev
```

Comment:
This is nicer to read here, but harder in the ocluster log.

Reply:
Nicer for humans, and nicer diffs. 
https://docs.docker.com/build/building/best-practices/#sort-multi-line-arguments